packages/xo-server/src/api/pool.mjs

```diff
@@ -138,14 +136,14 @@ export async function installPatches({ pool, patches, hosts }) {
     })
   }
 
-  await xapi.installPatches(opts)
+  await xapi.installPatches({ patches, hosts })
```

Comment:
I would prefer avoid using XO objects to XAPI methods, can we instead always pass `hosts`?

Reply:
That's what I understand too :thinking: